docs/get_started/plan_deployment/traditional_deployment/roadmaps/rm_install_deployment/rm_advanced_cfg/rm_production_farm.md

```diff
@@ -24,66 +24,55 @@ The topology includes a network load balancer to direct incoming traffic to the
 
 Gather information and software before you install HCL Digital Experience.
 
-1.  Check requirements.
+1.  Check requirements. See [Detailed system requirements](../../../../../system_requirements/index.md).
 
-    -   Documentation resource: [Detailed system requirements](../../../../../system_requirements/index.md)
+2.  Get required software. New and current users must [sign up or register at HCL Software](https://www.hcltechsw.com/wps/portal/about/welcome) and download their preferred HCL Digital Experience package.
 
-2.  Get the software. New and existing users need to [sign up or register at HCL Software](https://www.hcltechsw.com/wps/portal/about/welcome) and download their preferred HCL Digital Experience package
 
+## Installing prerequisite software
 
-## Installing prerequisite
+You can use currently installed prerequisite software. However, verify that your current version is supported. If it is not, upgrade to a supported version. Otherwise, install a web server, database server, and user registry server. Typically the database and user registry servers are already installed and configured. However, you might have to complete specific configuration steps that are required to integrate the software with the portal server. For moreinformation, see "Installing and preparing the prerequisite software."
```

Comment:
```suggestion
You can use currently installed prerequisite software. However, verify that your current version is supported. If it is not, upgrade to a supported version. Otherwise, install a web server, database server, and user registry server. Typically, the database and user registry servers are already installed and configured. However, you might have to complete specific configuration steps that are required to integrate the software with the portal server. For more information, see "Installing and preparing the prerequisite software."
```

Reply:
Yes